unison-cli/src/Unison/CommandLine/InputPatterns.hs

```diff
@@ -448,18 +451,22 @@ find' cmd global =
         [ ("`find`", "lists all definitions in the current namespace."),
           ( "`find foo`",
             "lists all definitions with a name similar to 'foo' in the current "
-              <> "namespace."
+              <> "namespace. Falls back to `find.all` if no matches are found."
           ),
           ( "`find foo bar`",
             "lists all definitions with a name similar to 'foo' or 'bar' in the "
               <> "current namespace."
           ),
+          ( "find.all foo",
+            "lists all definitions with a name similar to 'foo' in the current "
+              <> "namespace or direct dependencies."
```

Comment:
Doc nit: instead of

- "or direct dependencies"

I wonder if we should just explicitly call out the convention like,

- "including the `"lib"` namespace

Reply:
My doc changes aren't good, so nits are welcome. 

If we took this suggestion though, do you think we should also be explicit about not including transitive deps (`"lib".*."lib"`)?

Reply:
Oh yeah - well, that's getting awkward to word, isn't it 😭

Reply:
maybe `find foo bar` says `lists all definitions with a name similar to 'foo' or 'bar' (excluding those under 'lib')`
`find.all` says `lists all definitions with a name similar to 'foo' or 'bar' (including one level of 'lib')`

I think in the help text, we can just assume that people already know how ucm works, and we are just reminding them. We can go into more detail in tutorials or `helptopics`, etc.